databricks/koala/structures.py

```diff
@@ -156,6 +155,29 @@ def compute(self):
         """Alias of `toPandas()` to mimic dask for easily porting tests."""
         return self.toPandas()
 
+    @derived_from(pd.DataFrame)
+    def fillna(self, value=None, axis=None, inplace=False):
```

Comment:
Can you add also a docstring that mentions the limitations compared to pandas?
 - Series are not supported (yet) for values
 - not all the options are supported

Reply:
Done !